lua/plugins/dashboard.lua

```diff
@@ -52,7 +58,7 @@ preview = {
 	      key = '2',
 	      keymap = '',
 	      key_hl = 'Number',
-	      action = ':edit ~/.config/nvim/lua/config/plugins.lua'
+	      action = ':edit ~/.config/nvim/lua/config/plugins.lua' -- TODO: this needs tobe changed
```

Comment:
Agreed. I'll create an issue after the review and then start working on this aspect of the config as soon as possible.